tutorials/cloud-iot-rtdp/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Real time Data Processing With Google Cloud IoT
+description: Demonstrate how to build data processing backend for Google Cloud IoT.
+author: teppeiy
+tags: Cloud IoT Core, Cloud Dataflow, Cloud Functions, Java, Node.js
+date_published: 2017-11-29
+---
+# Real time Data Processing With Google Cloud IoT
+This repository contains sample code for Real time Data Processing With Google Cloud IoT solution. Sample MQTT client generates simulated data and submits them to Cloud IoT Core. Cloud Functions and Cloud Dataflow receive data through Cloud Pub/Sub.
+
+![Architecture](arch.png)
+
+## Prerequisites
+- Git
+- Google Cloud SDK
+- Java 1.8 or later
+- Apache Maven 3.3.x or later
+
+All commands in following steps are assumed to be executed under "community/tutorials/cloud-iot-rtdp" directory.
+
+## Enable APIs
+Enable following APIs from Google Cloud Console.
+- Cloud IoT Core
+- Cloud Functions
+- Cloud Dataflow
+
+## Create a Cloud Storage bucket and set environment variables.
+1. Create a Cloud Storage bucket and a temporary folder.
+2. Set environment variables. Corresponding resources are created in later steps.
+```bash
+% export PROJECT=<GCP Project ID>
```

Comment:
I suggest you provide actual names for people to use, as is becoming the standard in our tutorials.